yaramo/signal.py

```diff
@@ -52,8 +71,38 @@ class SignalState(Enum):
     hp2 = 2
     ks1 = 3
     ks2 = 4
-    sh1 = 5
-    sh2 = 6
+    sh0 = 5
```

Comment:
Are the values externally exposed? If yes, this is an incompatible change and we should increase the major version number. If not, we could consider using [enum.auto](https://docs.python.org/3/library/enum.html#enum.auto).

Reply:
I don' think so. `auto` is a nice idea!